src/azul/plugins/metadata/hca/transform.py

```diff
@@ -940,6 +943,32 @@ def _sequencing_input(self, sequencing_input: api.Biomaterial) -> MutableJSON:
             'sequencing_input_type': api.schema_names[type(sequencing_input)]
         }
 
+    @classmethod
+    def _aggregate_date_types(cls) -> FieldTypes:
+        return {
+            'document_id': null_str,
+            'submission_date': null_datetime,
+            'update_date': null_datetime,
+        }
+
+    def _aggregate_date(self) -> MutableJSON:
+        submission_date = None
+        update_date = None
+        for entity in self.api_bundle.entities.values():
+            # For submission dates we use a minimum value aggregation
+            if submission_date is None or entity.submission_date < submission_date:
+                submission_date = entity.submission_date
+            # For update dates we use a maximum value aggregation
+            if update_date is None or (
+                entity.update_date is not None and entity.update_date > update_date
+            ):
+                update_date = entity.update_date
```

Comment:
Use `min` and `max` built-ins with none-safe key. Not sure the comments are needed. They describe the how, which is obvious, not the why. Either remove or provide the underlying reason.

Reply:
I *think* we should exclude entities from stitched sub-graphs. Open to alternatives.